src/mixed-line-bar-chart/chart-popover.tsx

```diff
@@ -60,6 +63,7 @@ function MixedChartPopover<T extends ChartDataTypes>(
               onMouseLeave={onMouseLeave}
             >
               <ChartSeriesDetails details={highlightDetails.details} />
+              {footer && <InternalBox margin={{ top: 's' }}>{footer}</InternalBox>}
```

Comment:
Would it make sense to add the footer prop to the ChartPopover? This way the margin can be set once

Reply:
I've thought about it, but I think for now the content slot makes more sense. Because we're not actually introducing a formal visual "footer" concept (yet). Just like regular popovers dont really have a footer.